en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+
+## Step 5: Manage the API proxy
+
+Now that you have a tested API proxy, you can publish it and make it available for application developers to consume. In this guide, you will apply rate limiting to the API and publish it.
+
+### Step 5.1: Apply rate limiting to the API proxy
+
+1. In the left navigation menu, click **Deploy**.
+2. Go to the required environment card and click the settings icon corresponding to **API Configuration**.
+3. In the **API Configuration** pane, click **Rate Limiting** to expand the section.
+4. Select **API Level** as the **Rate Limiting Level**.
+5. Specify appropriate values for the **Request Limit** and **Time Unit** fields. You can proceed with the default values.
+6. Click **Apply**. This applies the rate limiting level to the API proxy and redeploys it.
+
+### Step 5.2: Publish the API proxy
+
+1. In the left navigation menu, click **Lifecycle** under **Manage**. This takes you to the **Lifecycle** page.
+2. Click **Publish**.
+3. In the **Publish API** dialog, click **Confirm** to proceed with publishing the API. If you want to change the display name, make the necessary changes and then click **Confirm**. This changes the API lifecycle state to **Published**.
+
+## Step 6: Invoke the API
+
+To generate credentials for the published API and invoke it via the Choreo Developer Portal, follow these steps:
+
+1. In the **Lifecycle** page, click **Go to Devportal**. This takes you to the `HR API` in the Choreo Developer Portal.
+
+2. **Generate Credentials**:
+    1. In the Developer Portal left navigation menu, click **Production** under **Credentials**.
+    2. Click **Generate Credentials**. Choreo generates new tokens and populates the **Consumer Key** and **Consumer Secret** fields.
+
+    !!! tip
+        To test the API via an API test tool or through code, click **Generate Access Token** and copy the test token. Alternatively, click **cURL** and copy the generated cURL command to use via a cURL client. You do not need to generate an access token if you are testing the API via the **Try Out** capability in the Choreo Developer Portal.
+
+3. **Invoke the API**:
+    1. In the Developer Portal left navigation menu, click **Try Out**.
+    2. In the **Endpoint** list, select **Development** as the environment to try out the API.
+    3. Click **Get Test Key** to generate an access token.
+    4. Expand the `GET /{departmentId}` resource and click **Try it out**.
+    5. Enter `1` as the **departmentId** and click **Execute**. You will see a response similar to the following:
+
+        ![Try out response](../../assets/img/develop-components/develop-a-rest-api-proxy/try-out-response.png){.cInlineImage-full}
+
+Now, you have gained hands-on experience creating, deploying, testing, and publishing an API proxy using Choreo API Manager.
+
+To learn more about the API management capabilities supported by Choreo API Manager, see [API Management](../../api-management/lifecycle-management.md).
```

Comment:
We are pointing to a wrong link it seems. We indicate about navigating to a generic feature page but we are directing to a lifecycle management page which is just a specific feature. If we don't have a better link to navigate, better to remove this line.

Reply:
Need to fix this in the original too.